app/assets/javascripts/goldencobra/html_editors/ckeditor/config.js

```diff
@@ -22,7 +22,7 @@ CKEDITOR.editorConfig = function(config) {
   // The toolbar groups arrangement, optimized for two toolbar rows.
   config.toolbarGroups = [
     { name: 'clipboard',   groups: [ 'clipboard', 'undo' ] },
-    { name: 'editing',     groups: [ 'find', 'selection', 'spellchecker' ] },
```

Comment:
Did not change this in the current work and did not remember, but it seems correct. No spellchecker wanted in ckeditor, a minimal setup is preferred.